README.md

```
@@ -1,8 +1,27 @@
+[![Go CI](https://github.com/vrnsky/golang-a-to-z/actions/workflows/test.yml/badge.svg)](https://github.com/vrnsky/golang-a-to-z/actions/workflows/test.yml)
+[![Go](https://img.shields.io/badge/Go-1.22-%2300ADD8.svg?&logo=go&logoColor=white)](#)
```

Markdownlint (0.37.0)

No empty links

(MD042, no-empty-links)